pontoon/base/views.py

```diff
@@ -407,20 +405,19 @@ def get_sibling_entities(request):
         .filter(order__lt=entities.order)
     )[:2]
 
-
-    print(entities)
-    print(preceding_entities)
-    print(succeeding_entities)
-
     entity = get_object_or_404(Entity, pk=entity)
     locale = get_object_or_404(Locale, code=locale)
     preferred_source_locale = ""
```

Comment:
I wasn't sure how you wanted me to handle the `preferred_source_locale`. I didn't know if you wanted me to copy exactly how the `entities()` view defines it. Let me know if I did it correctly :)